docs/dashboards/howto/components/custom.rst

```diff
@@ -117,3 +117,54 @@ You can then use the new component in your dashboard:
 
 .. image:: ../_images/components_gauge.png
    :alt: Form Filter
+
+
+Dynamic Data Filtering with GenericFilter
+
+Customize your Django Dashboards by integrating the GenericFilter component for dynamic data filtering. Here's how:
+
+1. Import GenericFilter:
+
+
+from dashboards.component.filters import GenericFilter
```

Comment:
Is this out of date? `GenericFilter` doesn't exist

Reply:
updated the documentation it was due to it being implemented this way before